examples/nbody/nbody.cpp

```diff
@@ -276,6 +276,9 @@ struct UpdateKernel
                 pos2 += threads
             )
                 temp( pos2 + threadBlockIndex ) = particles( start2 + pos2 + threadBlockIndex );
+
+            syncBlockThreads(acc);
```

Comment:
```suggestion
            syncBlockThreads( acc );
```

Reply:
Tbh I was wondering why this version worked at all without any `alpaka`namespaces (boy, I didn't use this kind of C++ for a way too long time, feel free to explain it too me :smile: ), but anyway: Imho the namespace of `syncBlockThreads` should be explicit: `alpaka::block::sync::syncBlockThreads( acc );`